src/services/checkpoints/CheckpointService.ts

```diff
@@ -246,15 +245,11 @@ export class CheckpointService {
 	}
 
 	public static async create({ taskId, git, baseDir, log = console.log }: CheckpointServiceOptions) {
-		git =
-			git ||
-			simpleGit({
-				baseDir,
-				binary: "git",
```

Comment:
I ran into issues on Windows when I specified `git` as the binary. Omitting it solved the problem. None of the other options are useful either, so they can be omitted.